src/main/kotlin/racingcar/RacingGame.kt

```diff
@@ -0,0 +1,16 @@
+package racingcar
+
+class RacingGame(private val cars: Cars) {
+    private val positions: MutableList<Positions> = arrayListOf()
+
+    constructor(cars: Int) : this(Cars(cars))
+
+    fun play(playCount: Int) {
+        for (i in 0 until playCount) {
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        repeat(playCount) {
```

Reply:
`StandardKt` 에 있었군요! 하나 또 배웁니다.
a65ff03a06d56c47bcc82fdf79c1036774e206dc 에서 반영하였습니다.